docs/source/getting_started/faq.rst

```diff
@@ -16,6 +16,38 @@ General
    JupyterLab? <https://jupyterlab.readthedocs.io>`__
 -  :ref:`How can you report a bug or issue? <issue>`
 
+Usage
+-----
+
+Notebook
+^^^^^^^^
+
+My notebook is displaying cell outputs in `iframe <https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe>`__. They are reset when scrolling back and forth.
+
+    Since JupyterLab v4, notebook rendering is optimized to display only the cells needed.
+    This has side effects with iframes.
+
+    The current workaround is to set the settings *Notebook* => *Windowing mode* to ``defer`` or ``none``.
```

Comment:
This brings the question: what default do we want?

Maybe the default should be `none`, and those who want performance could easily switch to `defer` via this settings?

Thinking that the majority of notebook users are dealing with small notebooks, and would not really benefit much from the performance improvements. And those who are dealing with large notebooks can then check the FAQ or the performance section of the docs to switch to `defer`.

Reply:
I'll open a new issue to discuss this and not block this PR.

Reply:
https://github.com/jupyterlab/jupyterlab/issues/14455